Docs update platform operator (#712)
meshcloud/meshcloud-docs: CONTRIBUTING.md

* A **User** is a person who uses the meshcloud panel to consume the meshcloud functionality, usually a developer or DevOps engineer. Users are focused on delivering value with their application deployments and use meshcloud to efficiently access, automate and administer the cloud technology they want to use. They want quick access to their deployment spaces and the resources they need. This means avoiding lenghty processes as much as possible, self-service and straight-through processing is key. Users usually work in teams which consists of colleagues of their own organizations or are mixed with external and internal specialists.
* A **Customer** is more a business-oriented role than a *user*. *Customers* have the responsibility for their users and projects in terms of data privacy, security, compliance, cost and capacity management (among others). Think of them as team leads or business managers/product owners with overall responsible for a business application.

> * **Partners** has an administrator role. He is resonsible for cloud management in organization by providing meshStack to *customers* and their *users*. Partners can access customers resources to support deployment of resources or to perform other administrative tasks such as cost control, cloud compliance, permossion management etc (some of these roles might be taken by different people than Partners, however we include them to the combined Partner role, as it is represents administrative role).. There is usually a contractual agreement between *partners* and *customers* that defines the obligations and rights of the *partner*.

Reviewer: ```suggestion
* **Partners** have an administrator role. They are responsible for cloud management in the organization by providing meshStack to their internal *customers* / *users*. Partners can access customers' resources to support the deployment of resources or to perform other administrative tasks such as cost control, cloud compliance, permission management, etc (some of these roles might be taken by different people than Partners, however, we include them to the combined Partner role, as it represents an administrative role)..There is usually a contractual agreement between *partners* and *customers* that defines the obligations and rights of the *partner*.
```

> * **Platform Operators** are also administrative role but with specific technical background and responsibilities for operating and providing cloud technology to *users*, which includes monitoring of cloud operations, intergating platforms, providing cloud services and ensure cloud compliance.

Reviewer: ```suggestion
* **Platform Operators** are also an administrative role but with specific technical backgrounds and responsibilities for operating and providing cloud technology to *users*, which include monitoring of cloud operations, integrating platforms, providing cloud services, and ensuring cloud compliance.
```

> All documentation is split between the User Docs and Administrator Docs. User Docs are more oriented toward basics of meshStack and could be valuble for all roles. Admistrator Docs are solely adressed to Partners and Platform Operators.

Reviewer: ```suggestion
All documentation is split between the User Docs and Administrator Docs. User Docs are more oriented toward the basics of meshStack and could be valuable for all roles. Admistrator Docs are solely addressed to Partners and Platform Operators.
```
